components/appointmentPicker/appointmentPickerSharedPropertiesService.js

```diff
@@ -8,6 +8,7 @@ define('appointmentPickerSharedPropertiesService', [], function() {
 			selectedDate: null,
 			selectedSite: null,
 			selectedSiteTitle: null,
+			isSelectedSiteVirtual: false,
```

Comment:
Why did you decide to initialize this as false?

Reply:
I guess it just seemed like a sane default for this--in most scenarios, I imagine the site will be physical rather than virtual. I just as easily could've done `null`, but I think since this one is a bit more deterministic, `false` suited better. But this is just me rationalizing it now--in the moment, I didn't think of any of this.